test/cpp/api/modules.cpp

```diff
@@ -1451,7 +1453,7 @@ TEST_F(ModulesTest, FunctionalWithTorchFunction) {
 
 TEST_F(ModulesTest, FunctionalArgumentBinding) {
   auto functional =
-      Functional(torch::elu, /*alpha=*/1, /*scale=*/0, /*input_scale=*/1);
```

Comment:
I guess this is why you work-chatted me about overloads being busted; not just `m()` stops working, `Functional()` *explicitly* stops working entirely (although, I probably would have suggested you rewrite these to pass in a lambda rather than casting to identify the overload instead). IDK man... I know you've invested a lot into this approach, but it really isn't a huge lift to just setup a completely new namespace, and the problems here fail the sniff test of being intrinsically related to the change you're trying to make (you're *just* trying to simplify codegen in autograd layer on redispatch and the sniff test says that unrelated UX shouldn't be a blocker here).

Reply:
yeah that's why I reached out to you in workchat, having noticed that Functional breaks too, it's suddenly worse than I thought. I'll introduce `_outf` overloads for now and investigate the struct idea I had to fix this more long term.